Market-Trading-System/src/main/java/DomainLayer/Market/User/IUserFacade.java

```diff
@@ -1,25 +1,25 @@
 package DomainLayer.Market.User;
 
 import DomainLayer.Market.Util.StorePermission;
-
-import javax.ws.rs.core.Response;
 import java.util.List;
 
 public interface IUserFacade {
     public String createGuestSession();
     public boolean login(String userName, String password) throws Exception;
     public void terminateGuestSession(String userName);
-    public void register(String userName, String password) throws Exception;
+    public void register(String userName, String password, int userAge) throws Exception;
     public void logout(String userName);
     public String viewShoppingCart(String userName);
     public void modifyShoppingCart(String userName);
     public void checkoutShoppingCart(String userName);
     public boolean checkPermission(String userName,long storeId, String permission);
     public void assignStoreOwner(String userName, long storeId);
-    public void assignStoreManager(String userName, long storeId);
+    public void assignStoreManager(String userName, long storeId, List<String> storePermissions);
     public List<String> getUserPermission(String userName,long storeId);
+    public boolean checkPermission(String userName);
+    public List<StorePermission> getUserPermission(String userName);
```

Comment:
remove duplicate of getUserPermission function, keep the one who return list<String> (I taked with amit about that)